calculator.rb

```diff
@@ -0,0 +1,78 @@
+###ARRAY DICTIONARY###
+acceptable_operation = %w(add + subtract - multiply * divide / exponents ^ remainder %)
+operation_map = {"add" => "+", "subtract" => "-", "multiply" => "*", "divide" => "/", "exponents" => "**", "remainder" => "%", "+" => "+", "-" => "-", "*" => "*", "/" => "/", "^" => "**", "%" => "%"}
```

Comment:
This is too long for 1-line.

```suggestion
operation_map = {
  "add" => "+", 
  "subtract" => "-", 
  "multiply" => "*", 
  "divide" => "/", 
  "exponents" => "**", 
  "remainder" => "%", 
  "+" => "+", 
  "-" => "-", 
  "*" => "*", 
  "/" => "/", 
  "^" => "**", 
  "%" => "%"
}
```